### ISSUE TITLE
Comparison fails with --visualMetricsPerceptual --visualMetricsContentful

### ISSUE THREAD

**Issue by Amerousful** [sitespeedio/sitespeed.io#4307]
### Have you read the documentation?

- [X] Yes, I've read the [how to make a reproducable bug guide](https://www.sitespeed.io/documentation/sitespeed.io/bug-report/)
- [X] Yes, I've read the [how to debug my script guide](https://www.sitespeed.io/documentation/sitespeed.io/scripting/#debug)

### URL

https://www.sitespeed.io/

### What are you trying to accomplish

Run with the comparing mode - https://www.sitespeed.io/documentation/sitespeed.io/compare/ 

### What browser did you use?

Chrome

### How to reproduce

```shell
Run twice: first to create the baseline output, then a second time to "activate" the comparison.

docker run --rm \
  --name sitespeed_test \
  -v "$(pwd):/sitespeed.io" \
  -v "$(pwd)/scripts:/scripts" \
  -v "$(pwd)/baseline:/baseline" \
  sitespeedio/sitespeed.io:35.3.0-plus1 \
  https://www.sitespeed.io/  \
  -n 1 \
  --compare.saveBaseline \
  --compare.id baseline-compare \
  --compare.baselinePath baseline \
  --visualMetrics --visualMetricsPerceptual --visualMetricsContentful
```

### Log output

```shell
ERROR: ExecaError: Command failed with exit code 1: python /usr/src/app/lib/plugins/compare/statistical.py

Traceback (most recent call last):
  File "/usr/src/app/lib/plugins/compare/statistical.py", line 33, in <module>
    stat, p = perform_test(test_type, metric_data['baseline'], metric_data['current'], **options)
  File "/usr/src/app/lib/plugins/compare/statistical.py", line 11, in perform_test
    if not has_variability(baseline) or not has_variability(current):
  File "/usr/src/app/lib/plugins/compare/statistical.py", line 7, in has_variability
    return len(set(sample)) > 1
TypeError: unhashable type: 'list'

Traceback (most recent call last):
  File "/usr/src/app/lib/plugins/compare/statistical.py", line 33, in <module>
    stat, p = perform_test(test_type, metric_data['baseline'], metric_data['current'], **options)
  File "/usr/src/app/lib/plugins/compare/statistical.py", line 11, in perform_test
    if not has_variability(baseline) or not has_variability(current):
  File "/usr/src/app/lib/plugins/compare/statistical.py", line 7, in has_variability
    return len(set(sample)) > 1
TypeError: unhashable type: 'list'
    at getFinalError (file:///usr/src/app/node_modules/execa/lib/return/final-error.js:6:9)
    at makeError (file:///usr/src/app/node_modules/execa/lib/return/result.js:108:16)
    at getAsyncResult (file:///usr/src/app/node_modules/execa/lib/methods/main-async.js:167:4)
    at handlePromise (file:///usr/src/app/node_modules/execa/lib/methods/main-async.js:150:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runStatisticalTests (file:///usr/src/app/lib/plugins/compare/helper.js:32:24)
    at async ComparePlugin.processMessage (file:///usr/src/app/lib/plugins/compare/index.js:192:29)
```

**Comment by Amerousful** [comment]
I'm not sure about the details of this exclusion, but we may also need to exclude:`visualMetricsPerceptual` and `visualMetricsContentful`
<img width="1708" alt="image 2093" src="https://github.com/user-attachments/assets/760a0874-ce5b-4a35-beaf-be253076a2ed">

**Comment by soulgalore** [comment]
Hi @Amerousful thanks for creating the issue! I have fixed it in main and will release it early next week when I'm back from vacation.

**Comment by Amerousful** [comment]
Yeah, I see that you're on vacation and that's why decided don't disturb you)
Have a good vacation!

**Comment by soulgalore** [comment]
This is fixed in 35.3.1!

**Comment by Amerousful** [comment]
@soulgalore Many thanks!)
